managers/templates/managers/projects_list.html

```diff
@@ -0,0 +1,209 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% block content %}
+{% load rest_framework %}
+{% load custom_tags %}
+
+<h1>Projects list
```

Comment:
If we are planning to use translations, we have add them in templates too. Please add them here and below.

Reply:
Done